src/IO/H5/StellarCollapseEos.cpp

```diff
@@ -0,0 +1,62 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "IO/H5/StellarCollapseEos.hpp"
+
+#include <algorithm>
+#include <array>
+#include <cstddef>
+#include <hdf5.h>
+#include <ostream>
+
+#include "ErrorHandling/Error.hpp"
+#include "IO/H5/CheckH5.hpp"
+#include "IO/H5/Helpers.hpp"
+#include "IO/H5/OpenGroup.hpp"
+#include "IO/H5/Type.hpp"
+#include "IO/H5/Wrappers.hpp"
+
+namespace h5 {
+StellarCollapseEos::StellarCollapseEos(bool exists, detail::OpenGroup&& group,
+                                       const hid_t location,
+                                       const std::string& /*name*/) noexcept
+    : exists_(exists),
+      group_(std::move(group)),
+      location_(location) {
+  if (not exists_) {
+    ERROR("Subfile does not exist");
```

Comment:
Could you print the name or something in the error? (If for some reason that's not possible, more info would be nice to have)